services/src/functional-tests/src/main/groovy/org/openkilda/functionaltests/helpers/FlowHelper.groovy

```diff
@@ -261,6 +262,28 @@ class FlowHelper {
         }
     }
 
+    /**
+     * Check that flow state is changed correctly after corresponding flow history action.
+     *
+     * @param flowId
+     * @param timeFrom history will requested from this timestamp
+     * @param event sequence of flow history actions
+     * @param state expected flow state
+     */
+    void verifyFlowState(String flowId, Long timeFrom, FlowHistoryEvent event, state) {
+        def timeTo = timeFrom ? System.currentTimeSeconds() : null
+        def flowHistory = northbound.getFlowHistory(flowId, timeFrom, timeTo).findAll {
+            it.action == event.initAction.toString()
+        }
+        assert flowHistory.size() == 1
```

Comment:
are you sure that you want to limit your method with this condition? What if there were more actions on the flow? I'm not insisting, but for me it kind of limits where it can be used or forces to use the 'from' timestamp. that's fine I guess, but requires a lot of 'timestamp' vars for long tests